### PR TITLE
feat(DTT-154): Remove pyinstaller workaround

### PR DESCRIPTION
**Pull request by ProfOak**
Prior to the recent Python3 upgrade we used an older version of the
requests library. This library came with a `cacert.pem` file needed to
make https connections. This file was separated out into it's own
library called `certifi`, and PyInstaller added a hook to include this
file on build time. The original workaround assumed the cacert.pem file
existed in the requests folder of the temp directory made by PyInstaller.

The workaround is no longer needed because of the PyInstaller hook.

Other changes in this commit revolve around Python3, and progressbar2 fixes.

- `e.message` is no longer needed in Python3.
- progress bar library was changed from progressbar to progressbar2 and
the behavior changed slightly.
- Reorganized imports in upload/client.py to pep8 recommendations.
- Added default parameter of True on verify request for download.

### CODE REVIEW

File: gdc_client/parcel/utils.py
Comment (kulgan) [inline gdc_client/parcel/utils.py]: Passing this in the constructor as `initial_value` might be more intuitive

Comment (ProfOak) [reply gdc_client/parcel/utils.py]: Thanks for pointing this out. There are apparently numerous parameters for this class constructor that do not show up in the documentation or examples on their website. I had to go into the source code to figure it out.

https://github.com/WoLpH/python-progressbar/blob/develop/progressbar/bar.py#L276-L280

**Review by kulgan — APPROVED** [review]
lgtm. @ProfOak do you mind updating the branch, seems out of date with the base

**Review by anmaxvl — APPROVED** [review]
can you squash commits before we merge?

**Comment by ProfOak** [comment]
> can you squash commits before we merge?

Correct me if I'm wrong but isn't there an option to do this on a merge? I'm not opposed to this. Please feel free to do this whoever has merge access. Let's get this merged soon, too. :-)

**Pull request merged**
